native/src/main/java/io/ballerina/stdlib/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -188,7 +188,7 @@ private void computeSourceContent(byte[] descriptor, String mode, boolean isRoot
                                         StandardDescriptorBuilder.getFileDescriptor(
                                                 StandardDescriptorBuilder.EMPTY_PROTO_PACKAGE_KEY).getMessageTypes()
                                                 .get(0).toProto(),
-                                        filePackage).build();
+                                        "", filePackage, StandardDescriptorBuilder.EMPTY_PROTO_PACKAGE_KEY).build();
                         messageList.add(message);
                         stubFileObject.addMessage(message);
                         hasEmptyMessage = true;
```

Comment:
Do we need to specially add an empty proto message here? Shall we handle the same way as wrapper types? Earlier we had to do the special case because we don't have a separate module. I think we avoid adding empty message type, in this case

Reply:
I think, if someone declared a proto message like below, we should generate a record instead of using a predefined wrapper.

```proto
message Empty {}
```

Reply:
AFAIK, it is not handled in this code section. it is handled like other custom message types. correct me if I am wrong

Reply:
+1, Now we can remove this special case for empty message types since we have a separate module now.